coalib/parsing/DefaultArgParser.py

```diff
@@ -187,6 +187,10 @@ def default_arg_parser(formatter_class=None):
         help='filters `--show-bears` by the given languages')
 
     outputs_group.add_argument(
+        '--filter-by', action='append', nargs='+',
+        help='filter --show-bears by the filter given as argument.')
```

Comment:
Though sometimes people don't use the feature when they don't know whether typing something wrong gives a better message (and I would count to those people).